diff --git a/dist/es/index.js b/dist/es/index.js
index ed51126cbb9d2977b94748c7e0b1793abf593c7c..613bc27b109faec6a0fc3e2a1514f319ed82f0a3 100644
--- a/dist/es/index.js
+++ b/dist/es/index.js
@@ -1,4 +1,5 @@
 import process from 'process';
+import { fileURLToPath } from 'url';
 import { isMainThread, parentPort, workerData, Worker } from 'worker_threads';
 import { isObject, hasOwnProperty, merge } from 'smob';
 import { minify } from 'terser';
@@ -118,7 +119,7 @@ class WorkerPool extends EventEmitter {
 
 function terser(options = {}) {
     const workerPool = new WorkerPool({
-        filePath: __filename,
+        filePath: fileURLToPath(import.meta.url),
         maxWorkers: options.maxWorkers
     });
     return {