diff --git a/lib/questions.js b/lib/questions.js
index 9275b1c53c8fc69d35d897f41f1cc3d70c69e868..91794e1c8ad6854c64deb6a093dce4bfeb063e31 100644
--- a/lib/questions.js
+++ b/lib/questions.js
@@ -116,6 +116,7 @@ module.exports = {
       {
         type: 'input',
         name: 'scope',
+        askAnswered: true,
         message: messages.customScope,
         when(answers) {
           return answers.scope === 'custom';